awx/api/pagination.py

```diff
@@ -3,14 +3,28 @@
 
 # Django REST Framework
 from django.conf import settings
+from django.core.paginator import Paginator as DjangoPaginator
 from rest_framework import pagination
+from rest_framework.response import Response
 from rest_framework.utils.urls import replace_query_param
 
 
+class DisabledPaginator(DjangoPaginator):
+
+    @property
+    def num_pages(self):
+        return 1
+
+    @property
+    def count(self):
+        return 200
```

Comment:
Many of the items in the UI only need like 5 entries. Giving such a large page size seems counter-productive. Does the page_size query param not work with this?

Reply:
I had to pick a value to make the paginator happy 😄, and this is the default page size.

Reply:
I want to make sure I am reading this right. So you can still get a list of 1 or 2 entries if desired?

Reply:
Yep:

`/api/v2/unified_jobs/?count_disabled=1&page_size=1` yields a `results` key with one item.

`/api/v2/unified_jobs/?count_disabled=1&page_size=5` gives you 5.